netci.groovy

```diff
@@ -235,6 +235,30 @@ commitPullList.each { isPr ->
   }
 }
 
+// True when this is a PR job, false for commit. We generally don't want to run the Loc status
+// check on PRs (as we don't want to block PRs due to unlocalized resources) and never want to
+// bother running it on feature branches.
+def locStatusCommitPullList = [false]
+if (branchName.startsWith("features/")) {
```

Comment:
Why not?

Reply:
I don't care if all the resources in feature branches are localized, so I may as well filter out the noise.